### PR TITLE
Git should ignore third_party/googletest

### PR DESCRIPTION
**Pull request by dneto0**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ compile_commands.json
 .ycm_extra_conf.py
 cscope.*
 third_party/glslang/
+third_party/googletest/
 third_party/gmock-1.7.0/
```
Comment (dmircevski) [inline .gitignore]: Is this line still necessary?

Comment (antiagainst) [inline .gitignore]: +1. The old gtest dir should be deleted.

Comment (dneto0) [inline .gitignore]: I intentionally left it in for the transition period where people will have both trees.  But I can see it's misleading.  Will remove it.

**Comment by antiagainst** [comment]
LGTM.

**Comment by dneto0** [comment]
rebased and pushed onto master

**Pull request closed without merging**